buildlib/jucx/jucx-build.yml

```diff
@@ -30,10 +30,65 @@ jobs:
       - bash: |
           set -eEx
           make -s -j`nproc`
-        displayName: Build ucx
+          source buildlib/az-helpers.sh
```

Comment:
Please verify this shell excerpt with shellcheck and fix findings

Reply:
We had issues with the newer command substitution syntax here. Prefer to leave it as is.